README.md

<!-- README.md is generated from README.Rmd. Please edit that file -->

# SMPD4

<!-- badges: start -->

[![Lifecycle:
stable](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg)](https://lifecycle.r-lib.org/articles/stages.html#stable)
<!-- badges: end -->

The goal of SMPD4 is to provide analysis ready data sets on SMPD4 data.

Sphingomyelin Phosphodiesterase 4 (SMPD4) is a type of sphinogmyelinase
protein encoded by the SMPD4 gene. Patients with a bi-allelic loss of
function variants in this gene are associated with a condition known as
neurodevelopmental disorder with microcephaly, arthrogryposis, and
structural brain anomalies (NEDMABA;
[618622](https://omim.org/entry/618622)).

Few cases exist in the scientific literature with [Magini et
al. (2019)](https://doi.org/10.1016/j.ajhg.2019.08.006) reporting 23
members of 12 unrelated families. The genotype-phenotype relationship of
NEDMABA is heterogeneous and aside from case-reports, little data is
available for statistical analysis. Magini et al. (2019) offer a
supplementary table with summarised clinical phenotype data of the
included subjects. Unfortunately these data are non standardised,
heterogeneous and largely textual based.

Further isolated examples exist in other papers and case studies, which
have also been included (Bijarnia-Mahay et al. (2022), Ji et al. (2022),
Monies et al. (2019), Ravenscroft et al. (2020)).

This project aims to standardise and process this data into tidy format
[(Wickham (2014))](https://doi.org/10.1016/j.ajhg.2019.08.006) so it is
analysis-ready and extensible for other researchers to better understand
the relationship between clinical factors discovered in the literature.

## Installation

You can install the development version of SMPD4 from
[GitHub](https://github.com/deanmarchiori/SMPD4) with:

``` r
# install.packages("devtools")
devtools::install_github("deanmarchiori/SMPD4")
```

## Example

``` r
library(SMPD4)

data("magini_2019")
head(magini_2019[,1:20])
#>    study         id family individual deceased survival_time variant_type
#> 1 Magini Magini_1_1      1          1     TRUE            30   homozygote
#> 2 Magini Magini_1_2      1          2     TRUE            60   homozygote
#> 3 Magini Magini_1_3      1          3     TRUE            30   homozygote
#> 4 Magini Magini_1_4      1          4     TRUE          1095   homozygote
#> 5 Magini Magini_2_1      2          1     TRUE          2190   homozygote
#> 6 Magini Magini_2_2      2          2    FALSE          1825   homozygote
#>   locus_1 locus_2 gender ethnicity consanguineous termination birth_gestation
#> 1    1320      NA female   Turkish           TRUE       FALSE              39
#> 2    1320      NA   male   Turkish           TRUE       FALSE              37
#> 3    1320      NA   male   Turkish           TRUE       FALSE              36
#> 4    1320      NA female   Turkish           TRUE       FALSE              38
#> 5    1483      NA female   Morocco           TRUE       FALSE              37
#> 6    1483      NA   male   Morocco           TRUE       FALSE              37
#>   route_vaginal_vs_c_sect birth_weight birth_ofc birth_length age_at_demise
#> 1                     SVD         2175      30.5           NA            30
#> 2                      CS         1860      31.0           NA            60
#> 3                      CS         1955      30.0           NA             2
#> 4                      CS         2780      30.5           41          1095
#> 5                      CS         2045      29.0           39          2190
#> 6                      CS         2300        NA           NA            NA
#>   age_at_last_follow_up
#> 1                    30
#> 2                    60
#> 3                    30
#> 4                  1095
#> 5                    NA
#> 6                  1825
```

## References

Bijarnia-Mahay, S., Somashekar, P. H., Kaur, P., Kulshrestha, S.,
Ramprasad, V. L., Murugan, S., … & Shukla, A. (2022). Growth and
neurodevelopmental disorder with arthrogryposis, microcephaly and
structural brain anomalies caused by Bi-allelic partial deletion of
SMPD4 gene. Journal of human genetics, 67(3), 133-136.

Magini, P., Smits, D. J., Vandervore, L., Schot, R., Columbaro, M.,
Kasteleijn, E., … & Mancini, G. M. (2019). Loss of SMPD4 causes a
developmental disorder characterized by microcephaly and congenital
arthrogryposis. The American Journal of Human Genetics, 105(4), 689-705.

Ji, W., Kong, X., Yin, H., Xu, J., & Wang, X. (2022). Case Report: Novel
Biallelic Null Variants of SMPD4 Confirm Its Involvement in
Neurodevelopmental Disorder With Microcephaly, Arthrogryposis, and
Structural Brain Anomalies. Frontiers in Genetics, 13.

Ravenscroft, G., Clayton, J. S., Faiz, F., Sivadorai, P., Milnes, D.,
Cincotta, R., … & Davis, M. R. (2021). Neurogenetic fetal akinesia and
arthrogryposis: genetics, expanding genotype-phenotypes and functional
genomics. Journal of medical genetics, 58(9), 609-618.

Monies, D., Abouelhoda, M., Assoum, M., Moghrabi, N., Rafiullah, R.,
Almontashiri, N., … & Alkuraya, F. S. (2019). Lessons learned from
large-scale, first-tier clinical exome sequencing in a highly
consanguineous population. The American Journal of Human Genetics,
104(6), 1182-1201.

Wickham, H. . (2014). Tidy Data. Journal of Statistical Software,
59(10), 1–23. <https://doi.org/10.18637/jss.v059.i10>

## Data use and licencing

These data are Copyright 2019 American Society of Human Genetics. Data
is used under Elsevier user license
<https://www.elsevier.com/about/policies/open-access-licenses/elsevier-user-license>

Use permitted for read, download, text and data mining. Used for
Non-Commercial purposes. All outputs of this R package are data mining
related for non-commerical use.


NEWS.md
# SMPD4 2.1.0

* Standardised formatting

# SMPD4 2.0.0

* added new datasets from other studies

# SMPD4 1.0.0

* Released stable version

# SMPD4 0.0.0.9000

* Added a `NEWS.md` file to track changes to the package.


data-raw/cadd_raw.R
## code to prepare `CADD` dataset goes here

CADD <- tibble::tribble(
  ~study,      ~family,   ~variant_type, ~cadd,
  "Magini",       "1",        "splice", 10.92,
  "Magini",       "2",        "splice",  29.4,
  "Magini",       "3",        "splice",  26.1,
  "Magini",       "3",      "nonsense",    36,
  "Magini",       "4",      "nonsense",    39,
  "Magini",       "5",      "deletion",    32,
  "Magini",       "6",        "splice",  26.2,
  "Magini",       "7",      "missense",  23.5,
  "Magini",       "7", "gene deletion",    NA,
  "Magini",       "8",      "nonsense",    37,
  "Magini",       "9",      "missense",  25.5,
  "Magini",       "10",      "deletion",    33,
  "Magini",       "11",      "missense",  26.2,
  "Magini",       "11",        "splice", 0.916,
  "Magini",       "12",      "deletion",  24.9,
  "Magini",       "12",        "splice",    26,
  "Monies_2019",  "1",        "splice",    NA,
  "Monies_2019",  "2",      "deletion",    NA,
  "Ravenscroft_2020", "1",      "missense",  22.8,
  "Ji_2022",       "1",      "nonsense",    NA,
  "Bijarnia-Mahay_2022", "1", "deletion", NA
) 

usethis::use_data(CADD, overwrite = TRUE)




data-raw/ji2022_raw.R
library(readr)

ji_2022 <- readr::read_csv('data-raw/ji_2022.csv', col_types = cols(
  study = col_character(),
  id = col_character(),
  family = col_character(),
  individual = col_factor(levels = c("1", "2", "3", "4"), ordered = FALSE, include_na = FALSE),
  deceased = col_logical(),
  survival_time = col_double(),
  variant_type = col_factor(levels = c("compound heterozygote", "homozygote"), ordered = FALSE, include_na = FALSE),
  locus_1 = col_double(),
  locus_2 = col_double(),
  gender = col_factor(levels = c("female", "male"), ordered = FALSE, include_na = FALSE),
  ethnicity = col_character(),
  consanguineous = col_logical(),
  termination = col_logical(),
  birth_gestation = col_double(),
  route_vaginal_vs_c_sect = col_factor(levels = c("CS", "SVD"), ordered = FALSE, include_na = FALSE),
  birth_weight = col_double(),
  birth_ofc = col_double(),
  birth_length = col_double(),
  age_at_demise = col_double(),
  age_at_last_follow_up = col_double(),
  microcephaly_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  sgp_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  iugr_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  talipes_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  rocker_bottom_feet_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  contractures_hands_deform_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  vocal_cord_palsy_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  arthrogryposis_contractures_other_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  hypercapnia_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  cleft_lip_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  inspiratory_stridor_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  polyhydramnios_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  coarse_facial_features_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  oligohydramnios_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  respiratory_distress_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  tracheostomy_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  prominent_ears_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  posteriorly_angulated_ears_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  prominent_philtrum_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  upper_lip_dysmorph_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  hypomyelination_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  cerebellar_hypoplasia_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  short_palpebral_fissures_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  epicanthal_folds_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  tube_feed_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  dysmorphic_features_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  patent_ductus_arteriosus_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  small_anterior_fontanel_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  receding_forehead_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  tracheo_laryngomalacia_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  gerd_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  thin_corpus_collosum_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  feeding_swallow_dysfxn_ind = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  seizures_age_started = col_character(),
  seizure_type_s = col_character(),
  eeg = col_character(),
  sleeping_problem = col_character(),
  behavioral_issues = col_character(),
  skeletal_changes_scoliosis = col_character(),
  hip_luxation = col_character(),
  development_head_control = col_character(),
  sitting_unsupported = col_character(),
  crawling = col_character(),
  walking = col_character(),
  speech = col_character(),
  iq_int_disability = col_character(),
  cp = col_character(),
  performance = col_character(),
  neuro_exam_eye_mvts_nystagmus = col_character(),
  strabismus = col_character(),
  feeding_swallow_dysfxn = col_character(),
  gag = col_character(),
  tone = col_character(),
  strength = col_character(),
  sensory_deficits = col_character(),
  abnormal_mvts = col_character(),
  dtr = col_character(),
  other_neuro_signs = col_character(),
  sensory_vision = col_character(),
  hearing = col_character(),
  other_organs = col_character(),
  endocrinology = col_character(),
  functional_tests = col_character(),
  metabolic_investigation = col_character(),
  cardiology = col_character(),
  seizure = col_double(),
  agenesis_of_corpus_callosum = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  abnormal_pattern_of_cerebral_sulci = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  left_sided_gallbladder = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  suspected_brain_stem_dysfunction = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  spastic_tetraparesis = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  severe_developmental_delay = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  acidosis = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  small_thorax = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  cyanosis = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  intubation = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  apnea = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  mr_is_reportedly_with_migrational_defect = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  at_birth_sga = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  epilepsy_and_apnea = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  choanal_atresia_surgeries_for_transposition_of_the_great_arteries = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  interventricular_and_interatrial_defects = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  smooth_philtrum = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  thin_lips = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  bilateral_simian_creases = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  low_anterior_hairline = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  short_philtrum = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  prominent_nose_bridge = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  depressed_nasal_bridge = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  prominent_suture_ridges = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  hypotelorism = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  hypertrichosis_of_lower_back = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  decreased_craniofacial_ratio = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  there_is_overlapping_of_the_sutures = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  bitemporal_narrowing = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  short_upturned_nose = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  short_neck = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  upward_sweep_of_hair = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  arched_eye_brows = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  bulbous_nose = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  prominent_lower_lip = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  broad_chin = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  prognathism = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  asd = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  left_multicystic_dysplastic_kidney = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  anteriorly_placed_anus = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  dilated_cardiomyopathy = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  cryptorchidism = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  hirsutism = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  intestinal_malrotation = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  dysphagia = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  intermittent_sinus_bradycardia = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  bilateral_staphylomas = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  lissencephaly = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  echogenic_choroid_plexus_on_head_us = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  head_us_mild_ventriculomegaly = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  brain_parenchyma_appears_homogeneous_without_evidence_of_focal_lesion = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  hypoplastic_cerebellum = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  borderline_small_brainstem = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  abnormal_cerebellar_folia = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  mild_vermian_hypoplasia = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  asymmetry_of_lateral_ventricles = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  lissencephaly_and_cerebellar_hypoplasia = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  signs_of_hypoxic_encephalopathy = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  mild_dilation_of_virchow_robins_spaces = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  pectus_excavatum = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  widely_spaced_nipples = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  respiratory_infections = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  enlarged_ventricles = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  eeg_normal = col_double(),
  bifid_uvula = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE),
  under_developed_cerebellar_inferior_vermis = col_factor(levels = c("0", "1"), ordered = FALSE, include_na = FALSE)
))



usethis::use_data(ji_2022, overwrite = TRUE)


R/data.R
#' SMPD4-Related Clinical Phenotype Data Set from Magini et al.(2019)
#'
#' A dataset of clical features and phenotypes of Neurodevelopmental Disorder with Arthrogryposis, Microcephaly and Structural Brain Anomalies Caused by Bi-allelic Partial Deletion of SMPD4 gene.
#' 
#' This data has been transformed into an analysis-ready tidy data set from Table S2 of Magini, Pamela, et al. "Loss of SMPD4 causes a 
#' developmental disorder characterized by microcephaly and congenital arthrogryposis." The American Journal of Human 
#' Genetics 105.4 (2019): 689-705.  
#'
#' @format A tibble: 21 x 152
#' 
#' \describe{
#'   \item{_ind}{columns ending in _ind are indicator columns created by merging duplicate tokenised terms into one feature.}
#' }
#' 
#' @source \url{https://doi.org/10.1016/j.ajhg.2019.08.006}
"magini_2019"

#' SMPD4-Related Clinical Phenotype Data Set from Bijarnia-Mahay et al. (2022)
#'
#' A dataset of clical features and phenotypes of Neurodevelopmental Disorder with Arthrogryposis, Microcephaly and Structural Brain Anomalies Caused by Bi-allelic Partial Deletion of SMPD4 gene.
#' 
#' This data has been transformed into an analysis-ready tidy data set from Bijarnia-Mahay, S., Somashekar, P. H., Kaur, P., Kulshrestha, S., Ramprasad, V. L., Murugan, S., ... & Shukla, A. (2022). Growth and neurodevelopmental disorder with arthrogryposis, microcephaly and structural brain anomalies caused by Bi-allelic partial deletion of SMPD4 gene. Journal of human genetics, 67(3), 133-136.  
#'
#' @format A tibble: 1 x 152
#' 
#' \describe{
#'   \item{_ind}{columns ending in _ind are indicator columns created by merging duplicate tokenised terms into one feature.}
#' }
#' 
#' @source \url{https://doi.org/10.1038%2Fs10038-021-00981-3}
"bijarnia_mahay_2022"

#' SMPD4-Related Clinical Phenotype Data Set from Ji et al. (2022)
#'
#' A dataset of clical features and phenotypes of Neurodevelopmental Disorder with Arthrogryposis, Microcephaly and Structural Brain Anomalies Caused by Bi-allelic Partial Deletion of SMPD4 gene.
#' 
#' This data has been transformed into an analysis-ready tidy data set from Ji, W., Kong, X., Yin, H., Xu, J., & Wang, X. (2022). Case Report: Novel Biallelic Null Variants of SMPD4 Confirm Its Involvement in Neurodevelopmental Disorder With Microcephaly, Arthrogryposis, and Structural Brain Anomalies. Frontiers in Genetics, 13.
#'
#' @format A tibble: 1 x 152
#' 
#' \describe{
#'   \item{_ind}{columns ending in _ind are indicator columns created by merging duplicate tokenised terms into one feature.}
#' }
#' 
#' @source \url{https://doi.org/10.3389%2Ffgene.2022.872264}
"ji_2022"

#' SMPD4-Related Clinical Phenotype Data Set from Monies et al. (2019)
#'
#' A dataset of clical features and phenotypes of Neurodevelopmental Disorder with Arthrogryposis, Microcephaly and Structural Brain Anomalies Caused by Bi-allelic Partial Deletion of SMPD4 gene.
#' 
#' This data has been transformed into an analysis-ready tidy data set from Monies, D., Abouelhoda, M., Assoum, M., Moghrabi, N., Rafiullah, R., Almontashiri, N., ... & Alkuraya, F. S. (2019). Lessons learned from large-scale, first-tier clinical exome sequencing in a highly consanguineous population. The American Journal of Human Genetics, 104(6), 1182-1201.
#'
#' @format A tibble: 2 x 152
#' 
#' \describe{
#'   \item{_ind}{columns ending in _ind are indicator columns created by merging duplicate tokenised terms into one feature.}
#' }
#' 
#' @source \url{https://doi.org/10.1016%2Fj.ajhg.2019.04.011}
"monies_2019"

#' SMPD4-Related Clinical Phenotype Data Set from Ravenscroft et al. (2020)
#'
#' A dataset of clical features and phenotypes of Neurodevelopmental Disorder with Arthrogryposis, Microcephaly and Structural Brain Anomalies Caused by Bi-allelic Partial Deletion of SMPD4 gene.
#' 
#' This data has been transformed into an analysis-ready tidy data set from Ravenscroft, G., Clayton, J. S., Faiz, F., Sivadorai, P., Milnes, D., Cincotta, R., ... & Davis, M. R. (2021). Neurogenetic fetal akinesia and arthrogryposis: genetics, expanding genotype-phenotypes and functional genomics. Journal of medical genetics, 58(9), 609-618.
#'
#' @format A tibble: 3 x 152
#' 
#' \describe{
#'   \item{_ind}{columns ending in _ind are indicator columns created by merging duplicate tokenised terms into one feature.}
#' }
#' 
#' @source \url{https://doi.org/10.1136%2Fjmedgenet-2020-106901}
"ravenscroft_2020"


#' Variant and CADD data
#'
#' Combined Annoation Dependent Depletion (CADD) data and Single Nucleotide Variant
#' type aggregated from studies on SMPD4 related variants.  
#' 
#' Data are manually aggregated from:  
#' 
#' Bijarnia-Mahay, S., Somashekar, P. H., Kaur, P., Kulshrestha, S., Ramprasad, V. L., Murugan, S., ... & Shukla, A. (2022). Growth and neurodevelopmental disorder with arthrogryposis, microcephaly and structural brain anomalies caused by Bi-allelic partial deletion of SMPD4 gene. Journal of human genetics, 67(3), 133-136.  
#' 
#' Magini, P., Smits, D. J., Vandervore, L., Schot, R., Columbaro, M., Kasteleijn, E., ... & Mancini, G. M. (2019). Loss of SMPD4 causes a developmental disorder characterized by microcephaly and congenital arthrogryposis. The American Journal of Human Genetics, 105(4), 689-705.  
#' 
#' Ji, W., Kong, X., Yin, H., Xu, J., & Wang, X. (2022). Case Report: Novel Biallelic Null Variants of SMPD4 Confirm Its Involvement in Neurodevelopmental Disorder With Microcephaly, Arthrogryposis, and Structural Brain Anomalies. Frontiers in Genetics, 13.  
#' 
#' Ravenscroft, G., Clayton, J. S., Faiz, F., Sivadorai, P., Milnes, D., Cincotta, R., ... & Davis, M. R. (2021). Neurogenetic fetal akinesia and arthrogryposis: genetics, expanding genotype-phenotypes and functional genomics. Journal of medical genetics, 58(9), 609-618.  
#' 
#' Monies, D., Abouelhoda, M., Assoum, M., Moghrabi, N., Rafiullah, R., Almontashiri, N., ... & Alkuraya, F. S. (2019). Lessons learned from large-scale, first-tier clinical exome sequencing in a highly consanguineous population. The American Journal of Human Genetics, 104(6), 1182-1201.  
#' 
#'
#' @format A tibble: 20 x 3
#' 
#' \describe{
#' \item{\code{family}}{Patient family identifier from study}
#'  \item{\code{variant_type}}{Type of SNV}
#'  \item{\code{cadd}}{Combined Annoation Dependent Depletion CADD score where available}
#' }
#' 
"CADD"